api/v1/policy_types.go

```diff
@@ -69,6 +69,12 @@ type PolicySpec struct {
 	// This provides the ability to enable and disable your policies.
 	Disabled bool `json:"disabled"`
 
+	// If set to true (default), all the policy's labels and annotations will be copied to the replicated policy.
+	// If set to false, only the policy framework specific policy labels and annotations will be copied to the
+	// replicated policy.
+	// +kubebuilder:validation:Optional
+	CopyPolicyMetadata *bool `json:"copyPolicyMetadata,omitempty"`
```

Comment:
Note that this is a pointer to avoid setting a default in the CRD to always cause this field to be shown when unset.